src/main/java/frc/trigon/robot/commands/factories/AmpCommands.java

```diff
@@ -0,0 +1,71 @@
+package frc.trigon.robot.commands.factories;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.trigon.robot.RobotContainer;
+import frc.trigon.robot.constants.AutonomousConstants;
+import frc.trigon.robot.constants.CommandConstants;
+import frc.trigon.robot.constants.FieldConstants;
+import frc.trigon.robot.subsystems.ampaligner.AmpAlignerCommands;
+import frc.trigon.robot.subsystems.ampaligner.AmpAlignerConstants;
+import frc.trigon.robot.subsystems.intake.IntakeCommands;
+import frc.trigon.robot.subsystems.intake.IntakeConstants;
+import frc.trigon.robot.subsystems.pitcher.PitcherCommands;
+import frc.trigon.robot.subsystems.pitcher.PitcherConstants;
+import frc.trigon.robot.subsystems.shooter.ShooterCommands;
+import frc.trigon.robot.subsystems.shooter.ShooterConstants;
+import frc.trigon.robot.subsystems.swerve.SwerveCommands;
+
+public class AmpCommands {
+    public static Command getAutonomousScoreInAmpCommand() {
+        return new ParallelCommandGroup(
+                getAutonomousPrepareForAmp(),
+                GeneralCommands.runWhenContinueTriggerPressed(getFeedToAmpCommand()),
+                getPathfindToAmpCommand().andThen(GeneralCommands.duplicate(CommandConstants.FACE_AMP_COMMAND))
+        );
+    }
+
+    public static Command getScoreInAmpCommand() {
+        return new ParallelCommandGroup(
+                getPrepareForAmpCommand(),
+                GeneralCommands.runWhenContinueTriggerPressed(getFeedToAmpCommand()),
+                GeneralCommands.duplicate(CommandConstants.FACE_AMP_COMMAND)
+        );
+    }
+
+    /**
+     * Creates a command that prepares all the components but only if it is close enough to the amp.
```

Comment:
What does "prepares all the components" mean? Why only if it's close to the amp? Why is this only used for auto? Is this for the 15 seconds auto routine?